core/templates/dev/head/pages/about/about.html

```diff
@@ -160,6 +160,14 @@
                 <a href="https://github.com/oppia/oppia/" class="inline-links">code</a> is released under an
                 <a href="https://www.apache.org/licenses/" class="inline-links">Apache 2.0</a> license.
               </p>
+              <div class="text-center">
+                  <button class="btn btn oppia-about-button" style="background-color: #009788; width:33%;">
```

Comment:
Where should I put that CSS class in this file, at the top or just after/before the buttons? And Should I specify comments for it?